.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+dist: trusty
+sudo: false
+
+git:
+    depth: 1
+
+php:
+  - 7.1
+  - 7.2
+
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+        - vendor
```

Comment:
I would not cache the vendor folder. This can make things really hard to debug if one of the build is corrupting it. Relying on the download cache is enough when using dist installations (which are not rate-limited anymore).